Client/Assets/SearchPlugins/bravesearch.xml

```diff
@@ -0,0 +1,18 @@
+<Url type="application/x-suggestions+json" method="GET" template="https://search.brave.com/api/suggest">
+  <Param name="q" value="{searchTerms}"/>
+  <Param name="rich" value="false"/>
+</Url>
+<Url type="text/html" method="GET" template="https://search.brave.com/search">
+  <Param name="q" value="{searchTerms}"/>
+  <Param name="source" value="ios"/>
+</Url>
+<SearchForm>https://www.google.com</SearchForm>
```

Comment:
Not sure what `SearchForm` is - but this is set to Google 😱

Reply:
That's a very good catch, copy/paste strikes again!